### ISSUE TITLE
Donation arrows doesn't change price

### ISSUE THREAD

**Issue by TomGUN02** [causefx/Organizr#1957]
###### Operating System: Unraid
<hr>

##### Problem Description:
In donations, If you change the amount with the arrows it doesn't change in the current donation.
BUT Manually typing the price will change the amount to donate.

<hr>

##### Reproduction Steps:
Go to donations, increase\decrease amount to donate with the arrows

<hr>

#### Errors on screen?  If so paste here:

```

```

**Comment by causefx** [comment]
here?
https://[captures.sonflix.com/DaXu5/mAPanEzI07.mp4/raw](https://captures.sonflix.com/DaXu5/mAPanEzI07.mp4/raw)

Sorry about the music

**Comment by tronyx** [comment]
Yes, but if you change the number and click the Donate button, the URL doesn't get updated with the amount, it just stays `1`.

**Comment by tronyx** [comment]
Bumping just to keep this active.
